articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+## Loading a Texture
+
+Loading textures, like any other asset in MonoGame, can be done by loading the asset file directly at runtime in game, or it can be loaded through the *content pipeline*.  Both of these methods are two sides of the same coin and there are trade offs to each approach.
+
+For instance, to load an image file directly at runtime, you would need to
+
+1. Add the image file to your project
+2. Configure the project to copy the image file to the game project build directory
+3. Load the image file as a texture at runtime using the [Texture2D.FromFile](xref:Microsoft.Xna.Framework.Graphics.Texture2D.FromFile(Microsoft.Xna.Framework.Graphics.GraphicsDevice,System.String)) method.
+
+When loading an image file as a texture directly like this, however, loads it in its compressed format such as *.png* or *.jpg*.  These compression formats are not understood by a Graphics Processing Unit (GPU); they will need to be decompressed into raw bytes as a format the GPU does understand before it can store the data.  Doing this can potentially leave a larger memory footprint for your assets.
+
+On the other side of this coin, MonoGame offers the *content pipeline*; a workflow for managing assets, such as image files.  The workflow is made up of a set of tools and utilities that are automatically added by default when you create a new MonoGame project using the MonoGame templates.  To use this workflow, you need to:
+
+1. Add the image file to your content project (*Content.mgcb* file) using the MGCB Editor.
+2. Perform a project build. Doing this, the *MonoGame.Content.Builder.Tasks* NuGet reference will compile the assets defined in the content project and automatically copy them to the game project build directory.
+3. Load the compiled asset at runtime using the [ContentManager](xref:Microsoft.Xna.Framework.Content.ContentManager).
+
+The following image illustrates this workflow:
+
+<figure><img src="./images/content-pipeline-workflow-full.png" alt="Figure 4-1: MonoGame Content Pipeline Workflow"><figcaption><p><strong>Figure 4-1: MonoGame Content Pipeline Workflow</strong></p></figcaption></figure>
+
+For the same amount of steps, you also get the benefit of the image file being pre-processed and compiled to an optimized format that can be understood by GPUs without needing to be decompressed first, reducing the memory footprint.  For instance, on desktops, image files can be compiled using [DXT compression](https://en.wikipedia.org/wiki/S3\_Texture\_Compression).
+
+> [!NOTE]

```

Comment:
In the build Phase, the text needs to be updated to say it is compiled "for the target platform", as the compilation will be different for some platforms.

Reply:
Step 2 updated to include the verbiage "for the target platform"